chapter_assets/email-templates/email-templates.md

```diff
@@ -0,0 +1,210 @@
+#Email templates
+
+## New Sponsor Information
+Hi XXX,
+
+I'm writing you to give you more detailed information about codebar.io and provide you with details for hosting a workshop.
+
+**About codebar**
+
+Codebar.io is an organization that started in the UK. The aim is to help women and other underrepresented groups in tech learn to program.
+
+We do this by pairing students with volunteer developers who work at different companies throughout Barcelona. We started a chapter in Barcelona over two years ago, and host workshops every two weeks. https://www.meetup.com/Codebar-Barcelona/
+
+Some of our students want to become full-time developers, and others are interested in learning the basics in a welcoming environment. Our community is expanding and we are looking for more companies to host us.
+
+**How to host a workshop**
+
+If you are interested in hosting a workshop, this is what we would need. A space that can host a minimum of 35 people. Some tables to work at, an internet connection, power outlets, or extension cables to plug in laptops, and since people are coming to the workshops after work we encourage hosts to provide food and drinks. We ask that vegan options be available as well.    
+
+The number of people that we open up the workshop to is **35 people and the workshop runs from 18:30-20:45**.
+
+
+
+**Typical Agenda for the event**
+
+- 18:00- 18:30 A member of the codebar team will arrive on the day of the event at 18h to help set up.
+
+- 18:30- 18:45 Participants arrive and network over drinks and food
+
+- 18:45- 19:00 Round of Presentations
+
+- 19:00- 19:10 Pairing of coaches and students
+
+- 19:10- 20:45 Coding  
+
+- 20:45 - 21:00 Wrap up and clean up
+
+
+**Additional information**
+
+Here is a link to more information about sponsoring. Do let me know if you have any further questions about how the event works.
+
+- http://manual.codebar.io/becoming-a-sponsor.html
+- http://manual.codebar.io/host-guide.html
+
+**Available dates**
+
+We are looking for hosts for the following dates:
+
+- YYYY.MONTH.DAY
+
+- YYYY.MONTH.DAY
+
+
+Let us know if any of these dates work for you. We look forward to hearing from you soon.
+
+Best,
+
+XXX
+
+## List of people attending
+
+Hi XXX,
+
+Here is the final list of people who will attend tomorrow's workshop.
+(paste link here)
+
+In total **XX people** signed up.
+
+Just a reminder, one of the codebar organizers will arrive at 18:00 to help set up for the event.  
+
+Let me know if you need anything else from us. Looking forward to seeing you and thank you for your support!  
+
+Best,
+
+XXX
+
+## Thank you for hosting codebar  
+
+Dear XXX,
+
+Thank you very much for hosting us yesterday and for the wonderful food and drinks. We really appreciate your hospitality. Hope you both have a great weekend and we look forward to working with you in the future.
```

Comment:
maybe we should have different templates for when the food and drinks were not to the level we expect. ( people should be able to have a dinner with what is provided, even if we don't need to be a dinner for everybody.